File: practices/time_of_day.py
# LC 7th Time Of Day


time = int(input("What is the time of day military time ther hour:\n"))

if time >= 17:
    print(f"Good evening it is {time}:00")
elif time>= 12:
    print(f"Good afternoon it is {time}:00")
else:
    print(f"good morning it is {time}:00")

File: c_programming/practices/time_of_day.c
// LC 7th Time Of Day
#include <stdio.h>
#include <string.h>

int main(void){
    int time;
    printf("What is the time of day military time ther hour:\n");
    scanf("%d", &time);

    if(time >= 17){
        printf("Good evening it is %d in military time", time);
    }else if(time >= 12){
        printf("Good afternoon it is %d in military time", time);
    }else{
        printf("good morning it is %d in military time", time);
    }
    return 0;
}